### ISSUE TITLE
Does not work on iOS

### ISSUE THREAD

**Issue by avatar** [kudago/smart-app-banner#33]
Hi,
I'm trying to implement this on my site. 
When I use force: 'ios', I see the banner like I should, but otherwise no banner is appearing :S

I also get no errors

Is this a known issue, or did I do something wrong?

Thanks!

**Comment by harlanrs** [comment]
In iOS 6 and greater the following meta tag is rendered by safari automatically on page load:
`<meta name="apple-itunes-app" content="app-id=[APPID]">`

Because of this automatic render behavior, this code base has a check for the version (to prevent duplicate banners for appearing):
`if (agent.os.name === 'iOS' && parseInt(agent.os.version) < 6) {`

this line will only be true and load their version of the banner if the user is on a device under iOS 6.

If you are adding the meta tag dynamically with javascript instead of manually inserting it in the head (as was the case with my project), safari won't register the meta tag on page load (because it doesn't exist immediately if it is being inserted by js) and won't show the native version of the banner. In this case, I was able to delete the `parseInt(agent.os.version) < 6` and have this library display the banner on any version of iOS (with the added benefit of working on chrome in iOS as well).

**Comment by marcofugaro** [comment]
Yup, this prevented me from using this plugin, the original one at least has a check 

https://github.com/jasny/jquery.smartbanner/blob/master/jquery.smartbanner.js#L35
